networkx/algorithms/connectivity/connectivity.py

```diff
@@ -78,9 +78,9 @@ def local_node_connectivity(
     cutoff : integer, float
         If specified, the maximum flow algorithm will terminate when the
         flow value reaches or exceeds the cutoff. This is only for the
-        algorithms that support the cutoff parameter: :meth:`edmonds_karp`
-        and :meth:`shortest_augmenting_path`. Other algorithms will ignore
-        this parameter. Default value: None.
+        algorithms that support the cutoff parameter: e.g., :meth:`edmonds_karp`,
```

Comment:
Didn't we decide that all the flow functions except `prepush_flow` supported `cutoff`?
In that case, maybe we can simplify these with something like:

``` 
    cutoff : integer, float, or None (default: None)
        If specified, the maximum flow algorithm will terminate when the
        flow value reaches or exceeds the cutoff. This doesn't work for the
        algorithm :meth:`prepush_flow` in which case `cutoff` is ignored.
```

Reply:
That was my first idea but then I thought what happens if there's some new flow function added that doesn't support cutoff.  Also maybe cutoff could be added to preflow push (I couldn't find anything about that online so I wanted to read the Cormen to fully understand the algorithm and decide if it could be possible to add without changing the algorithm itself). I thought this could be easier to maintain over time.  But I would be happy to change it if you think that's better. @dschult

Reply:
Yes -- I see the problem....      How about:

```
cutoff : integer, float, or None (default: None)
        If specified, the maximum flow algorithm will terminate when the
        flow value reaches or exceeds the cutoff. This only works for flows
        that support the cutoff parameter (most do) and is ignored otherwise.
```

Reply:
Perfect! I love that!